README.md

```diff
@@ -45,3 +45,9 @@ More details on contributing to the code are in the [CONTRIBUTING.md](CONTRIBUTI
 
 * Tom Hughes [@tomhughes](https://github.com/tomhughes/)
 * Andy Allan [@gravitystorm](https://github.com/gravitystorm/)
+
+# More Info
+
+* Translation is managed by [Translatewiki](https://translatewiki.net/wiki/Translating:OpenStreetMap). Most of the messages can be translated there. However, some translations come from external sources:
```

Comment:
This should better be integrated in existing reference in line 37

Reply:
Now Fixed @HolgerJeromin